.github/workflows/publish_dev_artifacts.yml

```diff
@@ -7,7 +7,11 @@ on:
   pull_request:
 
 env:
+  PRODUCT_NAME: kafka
   CARGO_TERM_COLOR: always
+  CARGO_INCREMENTAL: '0'
```

Comment:
I'm not sure we want to include those things. They come from here https://matklad.github.io/2021/09/04/fast-rust-builds.html and were a test to speed up our CI...not sure it helped much

Reply:
removed